actinia-alpine/Dockerfile

```diff
@@ -1,9 +1,11 @@
+FROM mundialis/actinia:alpine-dependencies-v1 as base
 FROM mundialis/grass-py3-pdal:releasebranch_8_0-alpine as grass
-FROM mundialis/esa-snap:s1tbx-8.0.3 as snap
-FROM mundialis/actinia-core:alpine-runtime-pkgs_v7
+FROM mundialis/esa-snap:s1tbx-d3eb736 as snap
```

Comment:
Why don't you use a tag here anymore?

Reply:
Would it make sense to go directly to grass 8.2 here?

Reply:
Because now a [pipeline](https://github.com/mundialis/esa-snap/blob/s1tbx/.github/workflows/docker.yml) is used to build and push the dockerimage. Before, a human beeing could determine the s1tbx version but it is not so easy to find it out for a machine. (It differs from general snappy version).